### PR TITLE
Fix remember me

### PR DESCRIPTION
**Pull request by abitme**

### CODE REVIEW

File: Models/IonAuthModel.php [hunk 1]
```diff
@@ -504,6 +504,11 @@ public function clearRememberCode(string $identity): bool
 			'remember_code'     => null,
 		];
 
+		// get identity of logged in user
+		if ($identity == $this->identityColumn) {
+			$builder = $this->db->table($this->tables['users']);
+			$identity = $builder->select($identity)->where('id', $this->session->get('user_id'))->get()->getRowArray();
```
Comment (benedmunds) [inline Models/IonAuthModel.php]: Why is this needed? We already have the identity and the user_id right?

Comment (abitme) [reply Models/IonAuthModel.php]: when logout $identity is equal to $this->identityColumn so it produces like this email => email therefore it can't update table users to clear remember code.  I add this line of code to produce like this email => example@mail.com so it can update table users, is that make sense?

Comment (benedmunds) [reply Models/IonAuthModel.php]: Could you do the update based on the user_id and remove one of the these queries?  Since the identify query is using the user_id anyway...

Comment (abitme) [reply Models/IonAuthModel.php]: If you want to remove these queries then we should remove calling clearRememberCode in logout because it pointless can't update users. Remember code in users table will be cleared when user logout and then login without check remember me. how about that?

Comment (shoyshai) [reply Models/IonAuthModel.php]: is the remember me working after doing this changes? Please update

Comment (abitme) [reply Models/IonAuthModel.php]: You can see all changes in here : https://github.com/benedmunds/CodeIgniter-Ion-Auth/pull/1475/files

Comment (shoyshai) [reply Models/IonAuthModel.php]: After making this changes still i remember me functionality is not working

**Comment by benedmunds** [comment]
Any updates here?

**Comment by benedmunds** [comment]
@shoyshai did you get this working?

**Comment by abitme** [comment]
> Any updates here?

I have made some update, please check it.

**Comment by shoyshai** [comment]
> 
> 
> @shoyshai did you get this working?

still remember me not working

**Comment by abitme** [comment]
> > @shoyshai did you get this working?
> 
> still remember me not working

it's working fine on my pc. Can you explain the issue why it's not working?

**Comment by benedmunds** [comment]
@shoyshai can you provide any more details?

**Comment by abitme** [comment]
@benedmunds have you checked my pull request? is there's something wrong?

**Comment by benedmunds** [comment]
I was hoping to hear back from @shoyshai before merging but this looks good to me. I'm going to merge and we can iterate later if any issues are surfaced.

Thanks for your time on this @abitme !
